server/sandbox_stop_linux.go

```diff
@@ -24,6 +24,9 @@ func (s *Server) stopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 		if err == sandbox.ErrIDEmpty {
 			return nil, err
 		}
+		if err == errSandboxNotCreated {
```

Comment:
Do we have evidence that we get a stop call before we return an id? The more likely scenario is a direct kill of processes  at the cgroup level by the kubelet.

Reply:
that's a good question. yeah it's not likely the kubelet can get that container....

(I need this PR independent of this race between kill and start btw, but I can put it with the timeout fix if it doesn't make sense alone)